Templates/Multiplayer/template.json

```diff
@@ -831,154 +807,6 @@
             "file": "Gem/Code/enabled_gems.cmake",
             "isTemplated": true
         },
-        {
-            "file": "Gem/Resources/GameSDK.ico",
-            "isTemplated": false
-        },
-        {
-            "file": "Gem/Resources/IOSLauncher/Images.xcassets/${Name}AppIcon.appiconset/Contents.json",
```

Comment:
+100 for red-code! How did you know to remove these IOSLauncher files?

Reply:
When I was pushing my project to git, git would fail to push with the clause that the files were corrupted. When I deleted, they had no lasting effect on the build or running of the project.

Reply:
From the naming of the folders I intuit that it was lingering files from someone's attempt at building the project for release on a target platform. Which I don't believe should be a part of the Template. But I may be wrong.